src/validators/schema.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require('joi');
+
+const signupSchema = Joi.object({
+    name: Joi.string().alphanum().min(3).max(30).required(),
+    email: Joi.string().email().required(),
+    password: Joi.string().min(6).required()
```

Comment:
I would recommend to add max limitation for password, because of [long password denial of service attacks](https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html#implement-proper-password-strength-controls)